gui/src/app/view/psbt.rs

```diff
@@ -572,12 +577,17 @@ pub fn inputs_and_outputs_view<'a>(
                         .style(theme::Button::TransparentBorder)
                     },
                     move || {
-                        coins
+                        tx.input
```

Comment:
Can we use `tx.input` also to get the number of coins spent above? For one of my PSBTs, it tells me "1 coin spent", but there are two inputs, one of which no longer exists.

Reply:
Also, should we use `if !tx.input.is_empty()` instead of `if !coins.is_empty()` above , as otherwise the inputs section will not be shown for a PSBT with only non-existent coins.